SimplenoteUITests/UIDs.swift

```diff
@@ -53,6 +53,10 @@ enum UID {
         static let cancel = "Cancel"
         static let dismissKeyboard = "Dismiss keyboard"
         static let deleteTagConfirmation = "Delete Tag"
+        static let select = "Select"
+        static let selectAll = "Select All"
+        static let deselectAll = "Deselect All"
+        static let trashIcon = "Trash Notes"
```

Comment:
Since we already have `itemTrash` and `trashNote` and `deleteNote` 🤦  buttons, what do you think of renaming this new button in this way:

```suggestion
        static let trashNotes = "Trash Notes"
```